metaflow/plugins/kfp/kfp.py

```diff
@@ -645,8 +645,10 @@ def _set_container_labels(
         # - In context of Zillow CICD self.username == "cicd_compile"
         # - In the context of a Zillow NB self.username == METAFLOW_USER (user_alias)
         # - In the context of Metaflow integration tests self.username == USER=$GITLAB_USER_EMAIL
-        user_email = self.username
-        container_op.add_pod_label("zodiac.zillowgroup.net/owner", user_email)
+        owner = self.username
+        if "@" in owner:
+            owner = owner.split("@")[0]
```

Comment:
nit: Can probably remove the `if`, it'll work regardless of whether the `@` is in the `owner` string :)

Reply:
ah true and cool!  thanks for sharing! this is more intentional and I think easier to read

Reply:
Cool either way works :)